test/integration/targets/rds_param_group/defaults/main.yml

```diff
@@ -0,0 +1,46 @@
+---
+rds_param_groups:
+  - name: rds-test-param-group-pg-96
```

Comment:
Sure, that makes sense. Is `resource_prefix` available out of the box or do I need to include something to make that work?